RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -1258,6 +1258,16 @@ boolean zoneItem(location loc);
 boolean set_property_ifempty(string setting, string change);
 boolean restore_property(string setting, string source);
 boolean clear_property_if(string setting, string cond);
+int fastenerCount();
+int lumberCount();
```

Comment:
Move these to the appropriate quest file please

Reply:
change made